packages/playground/windows/playground/HostingPane.xaml.cpp

```diff
@@ -428,6 +428,12 @@ void HostingPane::InitComboBoxes() {
   m_ReactAppNames = ref new Platform::Collections::Vector<String ^>();
 
   x_ReactAppName->ItemsSource = m_ReactAppNames;
+
+  try {
+    x_ReactAppName->IsEditable = true;
```

Comment:
Maybe add a comment for why you are doing this in code vs markup (I'm assuming this is a back compat thing?)